public/app/themes/clarity/inc/admin/wp-admin-bar.php

```diff
@@ -96,6 +88,23 @@ function agency_context_switcher_menu($wp_admin_bar)
     );
 }
 
+function get_agency_user_role()
```

Comment:
This is just a nit, but as there is nothing specific related to agency in the function, would `get_formatted_user_role` be a better fit?